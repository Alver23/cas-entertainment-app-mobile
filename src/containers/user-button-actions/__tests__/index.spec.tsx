// Dependencies
import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';

import UserButtonActions from '../index';

describe('<UserButtonActions />', () => {
  let component: ShallowWrapper;

  const props = {
    onEdit: jest.fn(),
    onDelete: jest.fn(),
  };

  beforeEach(() => {
    component = shallow(<UserButtonActions {...props} />);
  });

  afterEach(() => {
    jest.clearAllMocks();
  });

  it('should save a snapshot of the component', () => {
    expect(component).toMatchSnapshot();
  });
});
