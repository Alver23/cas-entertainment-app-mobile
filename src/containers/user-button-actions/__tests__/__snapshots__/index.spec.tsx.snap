// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UserButtonActions /> should save a snapshot of the component 1`] = `
<View
  style={
    Object {
      "flexDirection": "row",
      "justifyContent": "center",
      "marginVertical": 16,
      "width": "28%",
    }
  }
>
  <Component
    buttonStyle={
      Object {
        "marginRight": 8,
      }
    }
    iconConfig={
      Object {
        "name": "edit",
        "type": "feather",
      }
    }
    onPress={[MockFunction]}
    title="Editar"
  />
  <Component
    buttonStyle={
      Object {
        "marginLeft": 8,
      }
    }
    iconConfig={
      Object {
        "name": "close",
        "type": "material",
      }
    }
    onPress={[MockFunction]}
    title="Eliminar"
  />
</View>
`;
